x-pack/plugins/alerting/server/application/rule/transforms/transform_rule_attributes_to_rule_domain.ts

```diff
@@ -26,6 +26,9 @@ const INITIAL_LAST_RUN_METRICS = {
   total_alerts_detected: null,
   total_alerts_created: null,
   gap_duration_s: null,
+  // TODO: should initialised field, after inermidiate release
+  // gap_range: null,
+  // unfilled_gaps_ms: null,
```

Comment:
I'm not quite sure how these values are used, but I think initialising them here should be OK?

The intention with an intermediate release is to ensure that storage changes (incl. mappings) are made before code changes that depend on them.

In this case you are introducing a new _optional_ field. The optional part is what makes it OK to reference in code because we already treat it as if it may not be there.